### PR TITLE
fix(application request): add open overlay on progress button click

### PR DESCRIPTION
**Pull request by nidhigarg-bmw**
## Description

Add open overlay on progress button click

## Why

To open the overlay on button click for good user experience

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/740

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my own code
- [X] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/pages/Admin/components/RegistrationRequests/index.tsx [hunk 1]
```diff
@@ -76,13 +76,15 @@ export default function RegistrationRequests() {
   const [confirmCancelModalOpen, setConfirmCancelModalOpen] =
     useState<boolean>(false)
   const [selectedRequestName, setSelectedRequestName] = useState<string>('')
+  const [selectedActiveTab, setSelectedActiveTab] = useState<number>(0)
   const onTableCellClick = (params: GridCellParams) => {
     // Show overlay only when detail field clicked
-    if (params.field === 'detail') {
+    if (params.field === 'detail' || params.field === 'status') {
```
Comment (manojava-gk) [inline src/components/pages/Admin/components/RegistrationRequests/index.tsx:82]: Can you use enum here?

File: src/components/pages/Admin/components/RegistrationRequests/index.tsx [hunk 2]
```diff
@@ -76,13 +76,15 @@ export default function RegistrationRequests() {
...
       setSelectedRequestId(params.row.applicationId)
       setSelectedRequest(params.row)
       dispatch(fetchCompanyDetail(params.row.applicationId))
       setOverlayOpen(true)
+      setSelectedActiveTab(params.field === 'detail' ? 0 : 1)
```
Comment (manojava-gk) [inline src/components/pages/Admin/components/RegistrationRequests/index.tsx:87]: Can you use enum here?

**Review by manojava-gk — COMMENTED** [review]
Small findings. Rest looks good

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
Please add changelog

**Comment by nidhigarg-bmw** [comment]
> Please add changelog

No needed

File: src/components/pages/Admin/components/RegistrationRequests/index.tsx (hunk 1 above)
Comment (nidhigarg-bmw) [reply src/components/pages/Admin/components/RegistrationRequests/index.tsx]: @manojava-gk Made the changes

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Review by oyo — APPROVED** [review]

**Pull request merged**